### PR TITLE
Upgrade max supported PHP versions (8.2)

### PR DESCRIPTION
**Pull request by Altahrim**
Ref #294

### Content 
 - Upgrade max supported PHP (8.2)
 - Add and uses PHP CS fixer
 - Fix acceptance tests

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,17 +11,19 @@
...
     "scripts": {
-        "test": "phpunit -c tests/phpunit.xml",
-        "test:acceptance": "phpunit -c tests/phpunit.xml tests/Acceptance",
+        "cs:check": "php-cs-fixer fix --dry-run --diff",
+        "cs:fix": "php-cs-fixer fix",
+	"test": "phpunit -c tests/phpunit.xml",
+        "test:acceptance": "phpunit -c tests/phpunit.xml tests/Integration",
         "test:unit": "phpunit -c tests/phpunit.xml tests/Unit",
-        "test:acceptance:dev": "phpunit -c tests/phpunit.xml tests/Acceptance --no-coverage",
+        "test:acceptance:dev": "phpunit -c tests/phpunit.xml tests/Integration --no-coverage",
```
[the remark this answers is not in the document]
Comment (Altahrim) [reply composer.json:26]: I wasn't sure if used elsewhere or not. I'll change :)

File: appinfo/info.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	<category>security</category>
 	<bugs>https://github.com/ChristophWurst/twofactor_admin/issues</bugs>
 	<dependencies>
-		<php min-version="7.4" max-version="8.1"/>
+		<php min-version="8.0" max-version="8.2"/>
```
Comment (ChristophWurst) [inline appinfo/info.xml:24]: 7.4 must not be dropped before Nextcloud 25 support is gone too. Else the app store will send updates to Nextcloud 25 and PHP7.4 installations and they will :boom:

**Comment by ChristophWurst** [comment]
This repo uses conventional commits. Use `feat(deps): Add PHP8.0 support` and similar. In the end this will also automate the next release.

**Comment by Altahrim** [comment]
@ChristophWurst Do you mean scope is mandatory? 
Because except the fixup, theses are conventional commits :) 

> In the end this will also automate the next release

I also like the idea!

File: appinfo/info.xml (hunk 2 above)
Comment (Altahrim) [reply appinfo/info.xml:24]: Oh, I believed it will stop updates for instances not running PHP 8.0.
I'll fix this!

**Review by ChristophWurst — APPROVED** [review]
:+1: thanks

IMO `fix(tests): Fix integration tests` should be a chore to be hidden from the changelog

**Pull request merged**